sagemaker-remote-function/pytorch_mnist_sample_script/config.yaml

```diff
@@ -5,6 +5,6 @@ SageMaker:
       RemoteFunction:
         # role arn is not required if in SageMaker Notebook instance or SageMaker Studio
         # Uncomment the following line and replace with the right execution role if in a local IDE
-        RoleArn: arn:aws:iam::471094633457:role/SageMakerExecutionRole
+        # RoleArn: <replace the role arn here>
         InstanceType: ml.m5.xlarge
         Dependencies: ./requirements.txt
```

Comment:
Let's add 

```
IncludeLocalWorkDir: true
CustomFileFilter: 
    IgnoreNamePatterns:
        - "data"
    
```

Reply:
Oh somehow the change to that file didn't make it to the git update. Weird, I had that.